cinch/roles/jenkins_master/templates/enable-kerberos-sso.groovy

```diff
@@ -0,0 +1,21 @@
+def plugin = com.sonymobile.jenkins.plugins.kerberossso.PluginImpl.getInstance()
```

Comment:
Note that this (unreleased) change would help you to avoid the reflection madness: https://github.com/jenkinsci/kerberos-sso-plugin/commit/f1499b17e981ec7ac4c24598286720255daa312f. Would be `plugin.enabled = true`, etc.